spec/facility_spec.rb

```diff
@@ -16,11 +22,134 @@
+      camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+
+      facility_1.add_service('Vehicle Registration')
+      facility_1.register_vehicle(cruz)
+      expect(cruz.registration_date).to eq(Date.today)      
+    end
+
+    it "has a plate type" do
+      facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+
+      cruz.determine_plate_type
+      camaro.determine_plate_type
+      bolt.determine_plate_type
+
+      expect(cruz.plate_type).to eq(:regular)
+      expect(camaro.plate_type).to eq(:antique)
+      expect(bolt.plate_type).to eq(:ev)
+    end
+
+    it "has collection fees" do
+      facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+      cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+
+      facility_1.add_service('Vehicle Registration')
+      facility_1.register_vehicle(cruz)
+
+      expect(facility_1.collected_fees).to eq(100)
+
+      facility_1.register_vehicle(camaro)
+      facility_1.register_vehicle(bolt)
+
+      expect(facility_1.collected_fees).to eq(325)
+    end
+
+    it "#administer_written_test" do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 16 )
+      registrant_3 = Registrant.new('Tucker', 15 )
+      facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+
+      facility_1.add_service('Written Test')
+
+      expect(facility_1.administer_written_test(registrant_1)).to be true
+      expect(facility_1.administer_written_test(registrant_2)).to be true
+      expect(facility_1.administer_written_test(registrant_3)).to be false
+    end
+
+  it "#administer_road_test" do
```

Comment:
For these methods, we'd also want to test that the values stay "false" if the service of "Written Test" has not been added to the facility. Basically we want to test for each of those three conditions listed in the if statement for this method.